confluent_kafka/avro/__init__.py

```diff
@@ -98,22 +98,21 @@ def produce(self, **kwargs):
         value_schema = kwargs.pop('value_schema', self._value_schema)
         topic = kwargs.pop('topic', None)
         if not topic:
-            log.error("Topic name not specified.")
             raise ClientError("Topic name not specified.")
         value = kwargs.pop('value', None)
         key = kwargs.pop('key', None)
         if value:
             if value_schema:
                 value = self._serializer.encode_record_with_schema(topic, value_schema, value)
+                log.debug("Value encoded successfully!")
```

Comment:
I dont think there is much value (pun!) to these log messages, they dont include any actual information, and there is an exception to counter-represent their coverage, so I think you should remove all the just-before-exception logs, and logs like this.